src/stylesheets/_responsiveness.scss

```diff
@@ -71,16 +73,16 @@ Sample usage::
 ///  @include font(5vw, 35px, 150px, 50px);
 ///
 @mixin font($responsive, $min, $max: false, $fallback: false) {
-  $responsive-unitless: $responsive / ($responsive - $responsive + 1);
+  $responsive-unitless: math.div($responsive, ($responsive - $responsive + 1));
   $dimension: if(unit($responsive) == "vh", "height", "width");
-  $min-breakpoint: $min / $responsive-unitless * 100;
+  $min-breakpoint: math.div($min, $responsive-unitless * 100);
 
   @media (max-#{$dimension}: #{$min-breakpoint}) {
     font-size: $min;
   }
 
   @if $max {
-    $max-breakpoint: $max / $responsive-unitless * 100;
+    $max-breakpoint: math.div($max, $responsive-unitless * 100);
 
```

Comment:
If you test the app, you'll notice that responsiveness is breaking. I think the `* 100` should be in the numerator.